content/desktop/extensions/private-marketplace.md

```diff
@@ -0,0 +1,184 @@
+>
+> It's available to Docker Business customers only.
+{ .restricted }
+
+Learn how to configure and set up a private marketplace with a curated list of extensions for your Docker Desktop users.
+
+It is designed specifically for organizations who don’t give developers root access to their machines.
+
+Docker Extensions' private marketplace feature makes use of [Settings Management](../hardened-desktop/settings-management/_index.md) so admins have complete control over the private marketplace.
+
+## Prerequisites
+
+- [Download and install Docker Desktop 4.25.0 or later](https://docs.docker.com/desktop/release-notes/).
+- You must be an administrator for your organization.
+- You have the ability to push the `extension-marketplace` folder and `admin-settings.json` file to the locations specified below through device management software such as [Jamf](https://www.jamf.com/).
+
+## Step one: Initialize the private marketplace
+
+1. Create a folder locally for the content that will be deployed to your developers’ machines:
+
+   ```bash
+   mkdir myMarketplace
+   cd myMarketplace
+   ```
+
+2. Initialize the configuration files for your marketplace:
+
+   {{< tabs >}}
+   {{< tab name="Mac" >}}
+
+   ```bash
+   /Applications/Docker.app/Contents/Resources/bin/extension-admin init
+   ```
+
+   {{< /tab >}}
+   {{< tab name="Windows" >}}
+
+   ```bash
+   C:\Program Files\Docker\Docker\resources\bin\extension-admin init
+   ```
+
+   {{< /tab >}}
+   {{< tab name="Linux" >}}
+
+   ```bash
+   /opt/docker-desktop/extension-admin init
+   ```
+
+   {{< /tab >}}
+   {{< /tabs >}}
+
+This creates 2 files:
+
+- `admin-settings.json`, which activates the private marketplace feature once it’s applied to Docker Desktop on your developers’ machines
+- `extensions.txt`, which determines which extensions to list in your private marketplace.
+
+## Step two: Set the private marketplace behaviour
+
+The generated `admin-settings.json` file includes various settings you can modify.
+
+Each setting has a `value` that you can set, including a `locked` field that allows you to lock the setting and make it unchangeable by your developers.
+
+- `extensionsEnabled` enables Docker Extensions.
+- `extensionsPrivateMarketplace` activates the private marketplace and ensures Docker Desktop connects to content defined and controlled by the administrator instead of the public Docker marketplace.
+- `onlyMarketplaceExtensions` allows or blocks users from installing other extensions by using the command line. Teams developing new extensions must have this setting unlocked (`"locked": false`) to install and test extensions being developed.
+- `extensionsPrivateMarketplaceAdminContactURL` defines a contact link for users to request new extensions in the private marketplace. If `value` is empty then no link is shown to your developers on Docker Desktop, otherwise this can be either an HTTP link or a “mailto:” link. For example,
+
+   ```json
```

Comment:
oops, the ending "```" got stripped somehow

Reply:
the indentation looks off as well. should be two spaces

Reply:
> should be two spaces
Style guide states 3 https://docs.docker.com/contribute/style/formatting/#code-and-code-samples

Reply:
style guide is outdated then... iirc that had something to do w/ jekyll.